src/main/java/edu/pdx/cs/sftp/Client.java

```diff
@@ -579,40 +581,33 @@ void displayLogHistory() {
   }
 
   /**
-   * Deletes a file from the remote server. Can take one or multiple files in the format
-   * "testfile.txt, testfile2.txt"
+   * Deletes the specified file(s) from the remote server. Multiple file names can be included through a comma-separated
+   * list.
    *
-   * @param files -- The string read in main containing the names of the files.
+   * @param filename The string containing the name(s) of the file(s) to be deleted.
```

Comment:
```suggestion
   * @param filename the string containing the name(s) of the file(s) to be deleted.
```
[nit] the param descriptor should be lowercase.